### ISSUE TITLE
release-please releases do not trigger expected github actions workflows

### ISSUE THREAD

**Issue by ncouture-invue** [google-github-actions/release-please-action#402]
### TL;DR

Having setup release-please (`release-type: simple`) on a repository, when a release is created from a release-please PR, the release does not trigger other Github Actions.

For example I have configured the following workflow for troubleshooting and it is *not* executed when creating a release from a release-please PR (chore: release-x.x.x):

```
on:
  release:
    types:  [published, unpublished, created, edited, deleted, prereleased, released]
[...]
``` 

If what I describe here is indeed the normal behavior of release-please, it is not clear from the code or the documentation what event type a release-please release triggers (if any).

Are release-please releases not triggering any of the Github release-type events? 

### Expected behavior

any action configured to trigger on: release: types: ['created'] is triggered from a release-please release. 

### Observed behavior

Our debug action is never triggered.

### Action YAML

```yaml
- release-please.yaml -
on:
  push:
    branches:
      - main
name: release-please
jobs:
  release-please:
    runs-on: ubuntu-latest
    steps:
      - uses: GoogleCloudPlatform/release-please-action@v2
        with:
          release-type: simple
          changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"chore","section":"Miscellaneous","hidden":false}]'
---

- debug.yaml -
name: debug
on:
  release:
    types:  [published, unpublished, created, edited, deleted, prereleased, released]
jobs:
  output-debugging:
    name: Debug output
    runs-on: ubuntu-latest
    permissions:
      contents: 'read'
    steps:
    - id: debug
      name: Debug
      run: |-
        echo ${{ github.event_name }}
---
```

### Additional information

The overall use-case here is to build and publish the project's docker image on release creation.

The release-please `release-type` is `simple'.

**Comment by sethvargo** [comment]
I encountered this yesterday and we tracked down the root cause to be a GitHub "feature" (https://github.com/googleapis/release-please/issues/1142). In short, actions taken by the `GITHUB_TOKEN` in a workflow do not trigger additional workflows. There's more information and links to GitHub documentation in https://github.com/googleapis/release-please/issues/1142.

**Comment by xyy94813** [comment]
Hi @sethvargo ,

Can i resolve it by support a new token for `release-please-action`?
And, What is the minimal permission of new token?
